lib/hooks.dart

```diff
@@ -142,7 +134,7 @@ StateHook<T> useStateLazy<T>(T Function() init) => StateHook.lazy(init);
 /// ```
 ///
 /// See: <https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects>.
-void useEffect(dynamic Function() sideEffect, [List<Object> dependencies]) {
+void useEffect(dynamic Function() sideEffect, [List<Object?>? dependencies]) {
```

Comment:
Should `dependencies` be defaulted to `const []`?

I ask because I believe `useEffect` lints when `dependencies` is `undefined`.

Reply:
I don't think it does lint in that case; you can omit useEffect's dependencies if you want the effect to run after every render (or if you don't care if it runs after every render). 

See docs here: https://react.dev/reference/react/useEffect#useeffect:~:text=optional%20dependencies

Reply:
Hmmm... wonder where I got that from... nvm :)